modules/bridge/src/main/ts/ephox/bridge/api/Menu.ts

```diff
@@ -1,4 +1,5 @@
 import { ChoiceMenuItem, ChoiceMenuItemInstanceApi, ChoiceMenuItemSpec, createChoiceMenuItem } from '../components/menu/ChoiceMenuItem';
+import { CardMenuItemSpec, CardMenuItem, createCardMenuItem, CardMenuItemInstanceApi, CardItem } from '../components/menu/CardMenuItem';
```

Comment:
I noticed this when reviewing the mentions code, but this is the wrong location to export this IIUC. Any autocompleter items should be exposed via the `InlineContent` module, not menus.

Anything thing that's missing, is the specs haven't been exposed via the appropriate `Public` module in bridge (eg PublicMenu, PublicInlineContent, etc...).

Reply:
After having read more of the PR, it seems card is meant to be a new generic component for menu items. As such there's some other changes that need to be made:
- Needs adding to the relevant menu specs/logic (eg `NestedMenuItemContents`)
- `autocomplete` is a confusing property on something that's meant to be generic, so it needs a new name. I don't have any suggestions atm sorry.
- Needs to be added to the registry, as something that can be registered.
  - To match this, the autocompleter types need to handle a reference (name) and the logic in silver probably also needs to be updated to lookup names from the registry.
- `CardItemsSpec` probably should be exported, so it can be used when generating the card spec.